django/db/backends/mysql/base.py

```diff
@@ -360,11 +360,18 @@ def is_usable(self):
             return True
 
     @cached_property
-    def mysql_version(self):
+    def mysql_version_string(self):
         with self.temporary_connection() as cursor:
             cursor.execute('SELECT VERSION()')
-            server_info = cursor.fetchone()[0]
-        match = server_version_re.match(server_info)
+            return cursor.fetchone()[0]
+
+    @property
+    def mysql_version(self):
+        match = server_version_re.match(self.mysql_version_string)
         if not match:
-            raise Exception('Unable to determine MySQL version from version string %r' % server_info)
+            raise Exception('Unable to determine MySQL version from version string %r' % self.mysql_version_string)
         return tuple(int(x) for x in match.groups())
+
+    @property
+    def mysql_is_mariadb():
```

Comment:
Yeah, woops, I wrote this on a train without a working local mariadb so didn't run tests